### PR TITLE
Moved building information into documentation

### PR DESCRIPTION
**Pull request by sntdevco**
The documentation for building and installing has been moved into a new Documentation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,95 +31,15 @@ Install CMake: <https://www.cmake.org>
...
-```
-$ cmake -LH ..
-```
-
-### GUI
-
-CMake also provides a GUI that lets you specify the source directory and output (binary) directory
-that the build should be placed in.
-
+See [Documentation/Building#Unix](/Documentation/Building.md#building-on-unix) for more information
```
Comment (azat) [inline README.md]: This link is broken

Comment (sntdevco) [reply README.md]: Rebased and fixed

File: README.md [hunk 2]
```diff
@@ -84,95 +31,15 @@ Install CMake: <https://www.cmake.org>
      $ cmake -G "Visual Studio 10" ..   # Or whatever generator you want to use cmake --help for a list.
      $ start libevent.sln
 
+See [Documentation/Building#Windows](/Documentation/Building.md#building-on-windows) for more information
 ## CMake (Unix)
```
Comment (ploxiln) [inline README.md]: a blank line before headers would be good

Comment (sntdevco) [reply README.md]: @ploxiln Review the most recent commit

Comment (ploxiln) [reply README.md]: looks good to me, thanks

**Comment by sntdevco** [comment]
@azat Will you be merging this in?

**Comment by azat** [comment]
You mixed autoconf with cmake (`autogen.sh` note was under cmake build), so I rebased your patch a little (see d83b577d), objections?

**Comment by sntdevco** [comment]
> You mixed autoconf with cmake (`autogen.sh` note was under cmake build), so I rebased your patch a little (see [d83b577](https://github.com/libevent/libevent/commit/d83b577d2bc1fc917474d9b10dd7b3220f0dd8f6)), objections?

**Objection :hand:**
I have rebased and pushed, you don't need to rebase individually

**Comment by azat** [comment]
> Objection

What's the difference?

> I have rebased and pushed, you don't need to rebase individually

Information about flags for the autoconf still in the wrong section, plus I converted README to use ``` over `<tab>` for the markdown code blocks.

**Comment by sntdevco** [comment]
@azat Just a point -> you could've mentioned this earlier while I rebased to fix the links, but anyways I've made whatever changes you wanted and now it's safe to be merged as I have rebased the patches to reflect the same

**Comment by azat** [comment]
>Just a point -> you could've mentioned this earlier

I don't see any problems with me rebasing you patch (*you are still an author*).

And I gave you my patch/commit so that if you want to make this changes by your own, then you can take a look (*but looks like you didn't not look into it*), so *after 2 minutes of playing with links* here is how you can see it in github [web](https://github.com/libevent/libevent/compare/5e794493..d83b577d)

**Comment by sntdevco** [comment]
> > Just a point -> you could've mentioned this earlier
> 
> I don't see any problems with me rebasing you patch (_you are still an author_).
> 
> And I gave you my patch/commit so that if you want to make this changes by your own, then you can take a look (_but looks like you didn't not look into it_), so _after 2 minutes of playing with links_ here is how you can see it in github [web](https://github.com/libevent/libevent/compare/5e794493..d83b577d)

Sorry I didn't understand what you meant! I'll take care the next time. Anyways, all good and rolling?

**Comment by azat** [comment]
> I think this can be merged in now

If we are talking about d83b577d, then yep (since you had some disagreements, please confirm)

> all issues sorted?

Not all, but this is a good start

**Comment by sntdevco** [comment]
> > I think this can be merged in now
> 
> If we are talking about [d83b577](https://github.com/libevent/libevent/commit/d83b577d2bc1fc917474d9b10dd7b3220f0dd8f6), then yep (since you had some disagreements, please confirm)
> 
> > all issues sorted?
> 
> Not all, but this is a good start

Go on

**Comment by azat** [comment]
Applied, thanks!

**Pull request closed without merging**